src/app/assessment/assessment.component.ts

```diff
@@ -331,6 +331,21 @@ export class AssessmentComponent extends RouterEnter {
         }
       },
       (error) => {
+        if (!this.review.status) {
```

Comment:
This is still not the right place to put these codes. 
I'm working on [AV2-646] which will change the code structure of this part, so I think it's better to hold this story for now and go back to it after [AV2-646] has been merged.

Reply:
ok, sure. thanks @shawnm0705